frontend/components/organisms/QuestionDetail/index.tsx

```diff
@@ -80,6 +84,19 @@ const QuestionDetail = ({
                                 {views_count} {views_count > 1 ? 'times' : 'time'}
                             </span>
                         </div>
+
+                        {team_name !== '' ? (
+                            <div className="flex gap-1">
+                                <span>From Team </span>
+                                <span className="text-gray-500">{team_name}</span>
+                            </div>
+                        ) : (
+                            ''
+                        )}
+
+                        <div className="flex gap-1">
+                            {is_public ? <Icons name={'public'} /> : <Icons name={'private'} />}
+                        </div>
```

Comment:
Is the `div` necessary here?

Can you try this if it works?
```suggestion
                        <Icons name={is_public ? 'public' : 'private'} />
```